examples/flagsmith-on-eks/variables.tf

```diff
@@ -0,0 +1,10 @@
+variable "cluster_name" {
+  description = "Name of the EKS cluster Flagsmith gets deployed to."
+  type        = string
+}
+
+variable "db_password" {
```

Comment:
Should we use terraforms built ins to generate a password and store in an AWS secret? Maybe that's outside of the scope here.

Reply:
The truly right way to do this would be to utilise KMS, but it costs additional money and AFAIK not everyone chooses it for secret management. 

I could generate a random password instead of prompting for the value I suppose.

Reply:
Added an auto-generated passwords.

Stayed on a k8s secret because we want to avoid making vendor decisions here.